ops/ccs-ops-misc/cert-checker/cert-checker.sh

```diff
@@ -9,9 +9,9 @@ SEND_NOTIFICATIONS=${SEND_NOTIFICATIONS:-"false"}
 CRON_SCHEDULE="${SCHEDULE:-"0 12 * * *"}" # run at noon everyday
 
 # notifications
-WARN_WHEN="${WARN_WHEN:-"3000:4500"}"     # WARN when expiration is >= 30 but less than 45 days
-ALERT_WHEN="${ALERT_WHEN:-"1500:3000"}"   # expiration is >= 15 but less than 30 days
-PAGE_WHEN="${PAGE_WHEN:-"1:1500"}"        # expiration is >= 0 but less than 15 days
+WARN_WHEN="${WARN_WHEN:-"30:45"}"     # WARN when expiration is >= 30 but less than 45 days
+ALERT_WHEN="${ALERT_WHEN:-"15:30"}"   # expiration is >= 15 but less than 30 days
+PAGE_WHEN="${PAGE_WHEN:-"1:15"}"      # expiration is >= 0 but less than 15 days
```

Comment:
should this be 0:15?

Reply:
Refactored page logic to remove min date.